apps/meteor/app/api/server/v1/email-inbox.ts

```diff
@@ -25,40 +25,48 @@ API.v1.addRoute(
 	'email-inbox',
 	{ authRequired: true },
 	{
-		post() {
+		async post() {
 			if (!hasPermission(this.userId, 'manage-email-inbox')) {
 				throw new Error('error-not-allowed');
 			}
 			check(this.bodyParams, {
 				_id: Match.Maybe(String),
+				active: Boolean,
 				name: String,
 				email: String,
-				active: Boolean,
-				description: Match.Maybe(String),
-				senderInfo: Match.Maybe(String),
-				department: Match.Maybe(String),
+				description: String,
+				senderInfo: String,
+				department: String,
 				smtp: Match.ObjectIncluding({
-					password: String,
-					port: Number,
-					secure: Boolean,
 					server: String,
+					port: Number,
 					username: String,
-				}),
-				imap: Match.ObjectIncluding({
 					password: String,
-					port: Number,
 					secure: Boolean,
+				}),
+				imap: Match.ObjectIncluding({
 					server: String,
+					port: Number,
 					username: String,
+					password: String,
+					secure: Boolean,
 				}),
 			});
+			if (!this.bodyParams._id) {
```

Comment:
This endpoint serves for both update/creation. This validation will break it for creation path.